aspnetcore/signalr/configuration.md

```diff
@@ -272,24 +272,50 @@ HubConnection hubConnection = HubConnectionBuilder.create("https://example.com/c
 
 ### Configure timeout and keep-alive options
 
-Additional options for configuring timeout and keep-alive behavior are available on the `HubConnection` object itself:
+Additional options for configuring timeout and keep-alive behavior are available on <xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilder>:
 
 # [.NET](#tab/dotnet)
 
 | Option | Default value | Description |
 | ------ | ------------- | ----------- |
-| `ServerTimeout` | 30 seconds (30,000 milliseconds) | Timeout for server activity. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). This value must be large enough for a ping message to be sent from the server **and** received by the client within the timeout interval. The recommended value is a number at least double the server's `KeepAliveInterval` value to allow time for pings to arrive. |
+| `WithServerTimeout` | 30 seconds (30,000 milliseconds) | Timeout for server activity. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). This value must be large enough for a ping message to be sent from the server **and** received by the client within the timeout interval. The recommended value is a number at least double the server's keep-alive interval (`WithKeepAliveInterval`) value to allow time for pings to arrive. |
 | `HandshakeTimeout` | 15 seconds | Timeout for initial server handshake. If the server doesn't send a handshake response in this interval, the client cancels the handshake and triggers the `Closed` event (`onclose` in JavaScript). This is an advanced setting that should only be modified if handshake timeout errors are occurring due to severe network latency. For more detail on the handshake process, see the [SignalR Hub Protocol Specification](https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md). |
```

Comment:
This property is still only on `HubConnection`

Reply:
Thanks, I will move the detail of `HubConnection` object and `HubConnectionBuilder` to each property.

Reply:
Fixed in commit e99b087